ext/fluxnet_simulations/data_processing.jl

```diff
@@ -143,6 +157,32 @@ function get_data_at_start_date(
     return preprocess_func(v[idx_start])
 end
 
+"""
+    read_fluxnet_data(site_ID; hour_offset_from_UTC = nothing)
+
+Reads Fluxnet CSV data and processes timestamps, converting local time to UTC.
+Returns (data, columns, local_datetime, UTC_datetime).
+
+If `hour_offset_from_UTC` is nothing, the UTC datetime is not returned.
+"""
+function read_fluxnet_data(site_ID; hour_offset_from_UTC = nothing)
+    fluxnet_csv_path = ClimaLand.Artifacts.experiment_fluxnet_data_path(site_ID)
+    (data, columns) = readdlm(fluxnet_csv_path, ','; header = true)
+
+    # Convert timestamps
+    local_datetime = DateTime.(string.(Int.(data[:, 1])), "yyyymmddHHMM")
+
+    # Convert to UTC if hour offset is given
+    if isnothing(hour_offset_from_UTC)
+        UTC_datetime = nothing
+    else
+        UTC_datetime =
+            local_datetime .+ hour_offset_to_period(hour_offset_from_UTC)
+    end
+
+    return (data, columns, local_datetime, UTC_datetime)
```

Comment:
do we need both local and UTC date time? Shouldnt we do everything in UTC, always? Im not sure if the zenith angle computation will be correct if the simulation time is not in UTC.